VideoApi/VideoApi.UnitTests/Controllers/Conference/GetConferenceDetailsByIdTests.cs

```diff
@@ -1,35 +1,45 @@
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Net;
 using FluentAssertions;
 using Microsoft.AspNetCore.Mvc;
 using Moq;
 using NUnit.Framework;
-using VideoApi.DAL.Queries;
+using VideoApi.Contract.Requests;
 using Task = System.Threading.Tasks.Task;
 
 namespace VideoApi.UnitTests.Controllers.Conference
 {
-    public class GetConferenceDetailsByIdTests : ConferenceControllerTestBase
+    public class GetConferencesTodayByHearingVenueNameTests : ConferenceControllerTestBase
     {
+        private readonly  string _hearingVenueName = "MyVenue";
+        private readonly ConferenceForAdminRequest _conferenceForAdminRequest = new ConferenceForAdminRequest
+        {
+            HearingVenueNames = new List<string>(),
+        };
         [Test]
         public async Task Should_return_ok_result_for_given_conference_id()
         {
-            var result = await Controller.GetConferenceDetailsByIdAsync(TestConference.Id);
+
+            var hearingVenueNames = new List<string>(new string[] { _hearingVenueName });
+
+            _conferenceForAdminRequest.HearingVenueNames = hearingVenueNames;
+            var result = await Controller.GetConferencesTodayForAdminByHearingVenueNameAsync(_conferenceForAdminRequest);
 
             var typedResult = (OkObjectResult)result;
             typedResult.StatusCode.Should().Be((int)HttpStatusCode.OK);
```

Comment:
Nothing wrong with this just an FYI if you didn't know but a fluent assertion can be used here instead:
`result.As<OkObjectResult>().StatusCode.Should().Be((int) HttpStatusCode.OK);`

Reply:
Might also be worth asserting the value of the OkObjectResult is the expected value.